### DISCUSSION TITLE
OpenVPN Issue when sudo apt updating on Ubuntu 22.04 Jammy

### DISCUSSION THREAD

**Discussion by moayad-hsn** [OpenVPN/openvpn3-linux#227] [Q&A - Our support arena] [answered]
Hi,

I am using ubuntu 22.04 Jammy and trying to `sudo apt-get update`, while doing that I got this error :

`E: Failed to fetch https://packages.openvpn.net/openvpn3/debian/dists/DISTRIBUTION/InRelease  403  Forbidden [IP: 104.18.109.96 443]
E: The repository 'https://packages.openvpn.net/openvpn3/debian DISTRIBUTION InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'https://packages.openvpn.net/openvpn3/debian bookworm InRelease' does not seem to provide it (sources.list entry misspelt?)
`

I uninstalled OpenVPN using `apt-get purge openvpn` but the issu still exists.

Anyone has an idea on fixing this?

**Comment by dsommers** [comment, marked as answer]
~~I've alerted the ops team about this issue.  Hope this will be resolved quickly.~~

When looking more carefully at the error log ... here, reformatted:

      E: The repository 'https://packages.openvpn.net/openvpn3/debian DISTRIBUTION InRelease' is not signed.

The word `DISTRIBUTION` is wrong.  Please read the instructions in the URL below very carefully.  It says there what you need to do with that word.
https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux#Pre-builtpackages
